static/js/common/upload-addon.js

```diff
@@ -355,7 +358,7 @@
                         }
                       } else {  // This is a listed add-on.
                         if (automaticValidation && results.beta) {
-                          if (warnings > 0) {
+                          if (!passedValidation) {
```

Comment:
Same question as above, except this is confusing me because the comment above says it's for listed add-ons, but `passedValidation` only cares about the signing summary, shouldn't it depend on regular warnings as well ?